src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Main {
+
+    public static void main(String[] Args) {
+        final int ARRAY_LENGHT = 6;
```

Comment:
Constants in Java should be declared using `static final` modifiers and should be named using uppercase letters with underscores. Please correct the `ARRAY_LENGHT` [sic] constant (also there's a typo in the word 'LENGTH').